Core/Core/UIViews/PagesViewController.swift

```diff
@@ -51,6 +53,7 @@ public class PagesViewController: UIViewController, UIScrollViewDelegate {
         scrollView.showsVerticalScrollIndicator = false
         scrollView.canCancelContentTouches = true
 
+        currentWidth = scrollView.frame.inset(by: scrollView.adjustedContentInset).width
```

Comment:
Do we need this row? After `viewDidLoad` the `viewWillLayoutSubviews` should be called and the `currentWidth` will be re-calculated.

Reply:
It will only be reassigned if it's a different value, and that we want here.

Reply:
So if we remove this line and leave `currentWidth` at its starting value of 0 then it will get reassigned in `viewWillLayoutSubviews` since 0 will be different than the actual width. Would this have any side effects?

Reply:
Yes, the swipe animation would cancel out between module page changes.

Reply:
Okay, tested this, adjusting the scroll view on the first load won't affect the animations, amended code.